warehouse/templates/packaging/detail.html

```diff
@@ -98,10 +98,17 @@ <h1 class="package-header__name">
   </div>
 </section>
 
-{% if release.summary %}
+{% if release.summary %}<!-- todo: or logged in user can edit package -->
 <section class="horizontal-section horizontal-section--grey horizontal-section--thin">
   <div class="site-container">
-    <p class="package-description">{{ release.summary }}</p>
+    <div class="split-layout split-layout--middle">
+      {% if release.summary %}
+        <p class="package-description">{{ release.summary }}</p>
+      {% endif %}
+      <!-- todo: if logged in user can edit package -->
+        <a href="{{ request.route_path('manage.project.settings', name=project.normalized_name) }}" class="button button--highlight">Edit Project</a>
```

Comment:
If we really want an edit link on this page, we'll need to do a client-side include so the page doesn't require the use of a session (mostly adding this as a note to myself)